### PR TITLE
[OTS-694] Using OTCommonCommunicator enums that replace OTMultiPartyS…

### PR DESCRIPTION
**Pull request by southfox**
…creenSharerSignal and OTScreenShareSignal. (DRY principle)

Increment OTScreenShareKit to 2.0.0-beta5 and dependency OTAcceleratorPackUtil into 2.0.0-beta5.

This PR depends of the approval of OTAcceleratorPackUtil: opentok/acc-pack-common#65

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/opentok/screen-sharing-acc-pack/pull/30" target="_blank">ReviewNinja

**Comment by Lucashuang0802** [comment]
Don't need this for now.

File: ios/ScreenShareAccPackKit/OTScreenShareKit.podspec [hunk 1]
```diff
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
 
   s.source_files = 'ios/ScreenShareAccPackKit/OTScreenShareKit/**/*'
 
-  s.dependency 'OTAcceleratorPackUtil', '~> 2.0.0-beta4'
+  s.dependency 'OTAcceleratorPackUtil', '~> 2.0.0-beta5'
```
Comment (Lucashuang0802) [inline ios/ScreenShareAccPackKit/OTScreenShareKit.podspec]: Let's remove the version number and let it use the latest all the time 😃

File: ios/ScreenShareAccPackKit/OTScreenShareKit.podspec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "OTScreenShareKit"
-  s.version          = "2.0.0-beta4"
+  s.version          = "2.0.0-beta5"
```
Comment (Lucashuang0802) [inline ios/ScreenShareAccPackKit/OTScreenShareKit.podspec]: Let's do `2.0.0` for this. 😃

File: ios/ScreenShareAccPackKit/OTScreenShareKit/OTMultiPartyScreenSharer.h [hunk 3]
```diff
@@ -103,6 +87,11 @@ typedef void (^OTMultiPartyScreenSharerBlock)(OTMultiPartyScreenSharerSignal sig
  */
 @property (nonatomic) AVCaptureDevicePosition cameraPosition;
 
+/**
+ * Container of subscribers: instance of OTMultiPartyScreenShareRemote
+ */
+@property (readonly, nonatomic) NSMutableArray *subscribers;
```
Comment (Lucashuang0802) [inline ios/ScreenShareAccPackKit/OTScreenShareKit/OTMultiPartyScreenSharer.h]: Let's handle it the way how we handled acc-pack-comm

**Comment by Lucashuang0802** [comment]
Thanks for your sharp eyes. 👍

**Comment by southfox** [comment]
@Lucashuang0802 done.

**Pull request merged**